cohorts/views.py

```diff
@@ -1555,52 +1566,62 @@ def write(self, value):
 
 def streaming_csv_view(request, cohort_id=0):
```

Comment:
@s-paquette nothing jumped out but found this odd in accounts/views.py,  ln. 326:
```
if not user_id or not project_id or not project_name: 
    pass
else: 
```
why not just:
```
if user_id and project_id and project_name: 
```
this doesn't break anything, though